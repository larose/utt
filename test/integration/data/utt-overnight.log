2014-03-14 08:00 hello
2014-03-14 09:00 hard work

2014-03-17 09:00 overnight work
2014-03-17 10:15 hard work
