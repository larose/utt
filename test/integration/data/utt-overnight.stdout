
------------------------ Friday, Mar 14, 2014 (week 11) ------------------------

Working Time: 15h59
Break   Time: 0h00

----------------------------------- Projects -----------------------------------

(15h59) : hard work, overnight work

---------------------------------- Activities ----------------------------------

(1h00) : hard work
(14h59) : overnight work


----------------------------------- Details ------------------------------------

(1h00) 08:00-09:00 hard work
(14h59) 09:00-23:59 overnight work

